README.Mac.md

```diff
@@ -1,294 +1,289 @@
-```LoadModule php5_module libexec/apache2/libphp5.so```
-Ensure that the line is uncommented (remove the ```#``` at the beginning of the line)
+c. Paste the following into the ext-xdebug.ini file you just created (Note: replace the zend_extension path with the correct one on your machine):
+```
+[xdebug]
+zend_extension="/usr/local/Cellar/php@7.2/7.2.14/pecl/20170718/xdebug.so"
+xdebug.remote_enable=on
+xdebug.remote_host=localhost
+xdebug.remote_port=9000
+xdebug.show_error_trace=1
+xdebug.remote_autostart=1
+xdebug.remote_connect_back=0
+xdebug.idekey=phpstorm
+```
 
-## 3.3 Setup MySql with Apache2
-Do the following to allow for use of MySql with Apache2, unless link already exists
+#### Restart Apache with Xdebug configured.
 ```
-sudo mkdir /var/mysql
-sudo ln -s /tmp/mysql.sock /var/mysql/mysql.sock
+sudo apachectl -k restart
 ```
 
-## 3.4 Setting up user conf file
+### 1.7.2 Install [Xdebug toggle for OSX](https://github.com/w00fz/xdebug-osx) (recommended)
 
-In order for apache to know where to find the LORIS code base you have to first set up your user config file. To do so, open your user conf file using the following in the ``` /etc/Apache2/Users/``` directory:
+**a.** Install from the GitHub repository.
 
+**b.** Terminal command to view Xdebug state:
 ```
-sudo vi $username.conf
+xdebug
 ```
-<i>$username ⇾ your username for the computer</i>
-
-Once in the text editor, ensure that the file looks similar to the following:
 
+**c.** Terminal command to toggle xdebug (On or Off)
 ```
-<Directory "/Users/$username/Sites/">
-    Options Indexes MultiViews FollowSymLinks
-    AllowOverride None
-    Order allow,deny
-    Allow from all
-</Directory>
+xdebug on
+xdebug off
 ```
 
-**NOTE FOR OSX10/Apache 2.4 users:** 
+## 1.8 Install the database
 
-If the Order and Allow directives are not working then check if the module mod_access_compat is included in your apache configuration file. You can either include that module ( from usr/libexec/apache2 ) and continue to use the above configuration or include the new access control module mod_authz_host and use the following instead:
+We’re installing MariaDB (fork of mysql).
 
+**a.** Paste the commands into your terminal and hit enter:
+```
+brew update
+brew upgrade
+brew install mariadb
+brew services start mariadb
 ```
-<Directory "/Users/$username/Sites/">
-    Options Indexes MultiViews FollowSymLinks
-    AllowOverride None
-    Require all granted
-</Directory>
 
-```   
+**b.** Securing MySQL (set password for user “root” and disable outside localhost, remove anonymous users & test dbs):
+```
+mysql_secure_installation
+```
 
-The mod_access_compat module has been **deprecated** therefore it is suggested to use the new module and directives. 
+### 1.8.1 MySQL GUI tool (recommended)
 
-**END OF NOTE**
+[Sequel Pro](https://sequelpro.com/test-builds) (free)
 
+[TablePlus](https://tableplus.io) (free trial or pay for extra features)
```

Comment:
@driusan @maltheism Are these personnal choice or official LORIS recommendation?

Reply:
@PapillonMcGill they're recommendation from me and I think they should be included.

Reply:
Nothing about this document is an official LORIS recommendation, MacOS isn't supported.

Reply:
@maltheism please remove `(recommended)` on section 1.8.1 and 1.7.2. and replace by (optional).
on section 1.8.1, there should also be a line indicating that this list of tools is not exhaustive and represent only a few on the possible tools.

Reply:
@maltheism This still hasn't been fix

Reply:
@maltheism see 2 comment higher.